app/views/user_mailer/project_complete.html.slim

```diff
@@ -0,0 +1,51 @@
+p Hello #{@owner.display_name},
+
+p Wow! Your project is 100% transcribed!  What's next?
+ul
+  li Send a wrap up email to all of your collaborators announcing they did it! Remind them of the purpose of the project, thank them for their work, and direct them on to your next project or a colleague's project.
+  br
+  li 
+    'Export your transcriptions if you'd like. You can export your transcriptions in a variety of formats to use for other purposes, 
+    =link_to "here", collection_export_path(@collection.owner, @collection, only_path: false)
+    |. Or you can use the IIIF API.  If you need custom integration services to pull them into a downstream system, we can help - send us an email at 
```

Comment:
link IIIF API to here:  https://github.com/benwbrum/fromthepage/wiki/FromThePage-Support-for-the-IIIF-Presentation-API-and-Web-Annotations